build/Common.prod.props

```diff
@@ -1,13 +1,6 @@
 <Project>
   <Import Project=".\Common.props" />
 
-  <PropertyGroup>
-    <SignAssembly>true</SignAssembly>
```

Comment:
Are we not generating strong-named assemblies from this repo? If I recall, this can limit .NET Framework libraries that are strong-named from referencing the contrib instrumentation.

Reply:
oh... nevermind i see it's in the `Common.props`

Reply:
Ya this was just a bit of cleanup there was some duplication in there.